File: Examples/VisualBasic/ProgrammersGuide/Working-Diagrams/ExportToHTML.vb
'////////////////////////////////////////////////////////////////////////
' Copyright 2001-2013 Aspose Pty Ltd. All Rights Reserved.
'
' This file is part of Aspose.Diagram. The source code in this file
' is only intended as a supplement to the documentation, and is provided
' "as is", without warranty of any kind, either expressed or implied.
'////////////////////////////////////////////////////////////////////////

Imports Microsoft.VisualBasic
Imports System.IO

Imports Aspose.Diagram

Namespace VisualBasic.Diagrams
    Public Class ExportToHTML
        Public Shared Sub Run()
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_Diagrams()

            Dim diagram As New Diagram(dataDir & "ExportToHTML.vsd")

            diagram.Save(dataDir & "outputVSDtoHTML.html", SaveFileFormat.HTML)
        End Sub
    End Class
End Namespace

File: Examples/CSharp/ProgrammersGuide/Working-Diagrams/ExportToHTML.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2015 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Diagram. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Diagram;

namespace CSharp.Diagrams
{
    public class ExportToHTML
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_Diagrams();
            
            Diagram diagram = new Diagram(dataDir + "ExportToHTML.vsd");

            diagram.Save(dataDir + "outputVSDtoHTML.html", SaveFileFormat.HTML);
        }
    }
}